src/oas.md

```diff
@@ -1102,24 +1106,39 @@ description: username to fetch
 required: true
 schema:
   type: string
+examples:
+  "Edsger Dijkstra":
+    dataValue: edijkstra
+    serializedValue: edijkstra
+  Diṅnāga:
+    dataValue: diṅnāga
+    serializedValue: di%E1%B9%85n%C4%81ga
+examples:
+  Al-Khwarizmi:
+    dataValue: "الخوارزميّ"
+    serializedValue: "%D8%A7%D9%84%D8%AE%D9%88%D8%A7%D8%B1%D8%B2%D9%85%D9%8A%D9%91"
 ```
 
-An optional query parameter of a string value, allowing multiple values by repeating the query parameter:
+An optional query parameter of a integer value, allowing multiple values by repeating the query parameter
```

Comment:
```suggestion
An optional query parameter of a string value, allowing multiple values by repeating the query parameter
```

Reply:
Ugh... ever time I redo this part of a PR (which I've done like 4 times now as these examples have been impacted by multiple other PRs that have multiple revisions) I think I have gotten the type mis-aligned one way or another.  Thanks for catching!